chain/x/zoracle/handler.go

```diff
@@ -215,10 +216,27 @@ func handleMsgRequestData(ctx sdk.Context, keeper Keeper, msg MsgRequestData) sd
 }
 
 func handleMsgReportData(ctx sdk.Context, keeper Keeper, msg MsgReportData) sdk.Result {
+	startGas := ctx.GasMeter().GasConsumed()
+
+	// Save new report to store
 	err := keeper.AddReport(ctx, msg.RequestID, msg.DataSet, msg.Sender)
 	if err != nil {
 		return err.Result()
 	}
+
+	// Refund the reporter
+	err = keeper.SupplyKeeper.SendCoinsFromModuleToAccount(
+		ctx,
+		auth.FeeCollectorName,
+		msg.GetSigners()[0],
+		sdk.NewCoins(
+			sdk.NewCoin("uband", msg.RefundGasPrice.AmountOf("uband").Mul(sdk.NewDec(int64(ctx.GasMeter().GasConsumed()-startGas))).TruncateInt()),
```

Comment:
Can we use msg.RefundGasPrice as gasPrice and use this [function](https://github.com/cosmos/cosmos-sdk/blob/master/types/dec_coin.go#L207) convert to Coins?